src/screens/SendToken/SendTokenPinConfirmScreen.js

```diff
@@ -58,9 +62,10 @@ class SendTokenPinConfirmScreen extends React.Component<Props, State> {
   handleTransaction = (pin: string, wallet: Object) => {
     const { sendAsset } = this.props;
     const { transactionPayload } = this.state;
-    this.setState({
-      isChecking: true,
-    }, () => sendAsset(transactionPayload, wallet, this.handleNavigationToTransactionState));
+    this.setState({ isChecking: true }, () => {
+      Answers.logCustom('Send Transaction', { Source: this.source });
```

Comment:
Also, for this - should we track transaction attempts or track only successful transactions?
At this point transactions can fail.

Reply:
We wanted to have on one side the `Transaction sent` event, and in the other side the `Transaction success`/`Transaction failed`(I'm adding those in another PR), but now that you mention, I'm not sure if that makes sense or maybe we could use another name for this event. I'm gonna double check that.

Assuming we want to have the send and success/fail in separate events, do you think this would be the right place for this one?

Reply:
Yep, If you could also want to track transaction attempts - this method would suit you well :)

Reply:
Thanks, then we're gonna have this event as a transaction start, with the source information. And then have a fail/success event.